bitbucket.md

```diff
@@ -0,0 +1,39 @@
+# Bitbucket
+
+1. Install Bitbucket
+ - https://www.atlassian.com/software/bitbucket/download
+
+1. Install the following BitBucket Server plugin by logging into the web GUI of the Stash server and going to `Find new add-ons`.
+  * https://marketplace.atlassian.com/plugins/com.atlassian.stash.plugin.stash-web-post-receive-hooks-plugin/server/overview
+  * You can also install it manually by downloading the jar (https://confluence.atlassian.com/display/UPM/Installing+add-ons)
+
+1. Make a `Project` called `puppet` (with a short name of `PUP`)
+
+1. Create a repository called `control-repo`
+
+1. * Create a user called `r10k` with a password of `puppet`.
+   * Make the r10k user an admin of the `PUP` project.
+
+1. Either use the admin user to test pushing code, or create a user for yourself and add your SSH key to that user.
+  * If making a user for yourself, give your user account read/write or admin privilege to the `PUP` project.
+
+1. `git mv hieradata/nodes/example-puppet-master.yaml hieradata/nodes/<fqdn_of_your_puppet_master>.yaml`
```

Comment:
I was under the impression that the webhook part of the gms module doesn't work with stash because of the need for the plugin.  

As well as needing to add the Puppet CA as a trsuted ca in the java keystore for bitbucket. 

Where do those instructions come into play?